toolchain/semantics/semantics_handle_if_expression.cpp

```diff
@@ -25,20 +26,20 @@ auto HandleIfExpressionIf(Context& context, ParseTree::Node if_node) -> bool {
   return true;
 }
 
-auto HandleIfExpressionThen(Context& context, ParseTree::Node then_node)
+auto HandleIfExpressionThen(Context& context, ParseTree::Node parse_node)
     -> bool {
   // Convert the first operand to a value.
   auto [then_value_node, then_value_id] =
       context.node_stack().PopExpressionWithParseNode();
   context.node_stack().Push(then_value_node,
                             context.ConvertToValueExpression(then_value_id));
 
-  context.node_stack().Push(then_node, context.node_block_stack().Pop());
+  context.node_stack().Push(parse_node, context.node_block_stack().Pop());
```

Comment:
This change is understandable but unfortunate -- there are multiple different parse nodes in scope here, so having descriptive names for the `then_node` versus the `then_value_node` seems useful. Same thing happens in the next function, where we have an `if_node`, a `then_node`, and an `else_node` in scope before this change, which seems easier to understand than calling one of the three `parse_node`.

Is there some way we can turn off this checking for the `Handle` functions? If so, do you think we should?

Reply:
Sounds like you have a stronger preference than me. Trying to figure out a solution, NOLINTNEXTLINE and NOLINTBEGIN didn't seem to suppress this warning the way I'd expect, so how about this "alias" approach?